livekit-plugins/livekit-plugins-spitch/livekit/plugins/spitch/tts.py

```diff
@@ -0,0 +1,96 @@
+import uuid
+from dataclasses import dataclass
+
+import httpx
+
+import spitch
+from livekit.agents import (
+    DEFAULT_API_CONNECT_OPTIONS,
+    APIConnectionError,
+    APIConnectOptions,
+    APIStatusError,
+    APITimeoutError,
+    tts,
+)
+from spitch import AsyncSpitch
+
+SAMPLE_RATE = 24_000
+NUM_CHANNELS = 1
+MIME_TYPE = "audio/wav"
+
+
+@dataclass
+class _TTSOptions:
+    language: str
+    voice: str
+
+
+class TTS(tts.TTS):
+    def __init__(self, language: str, voice: str):
```

Comment:
Can we provide good defaults?
```suggestion
    def __init__(self, *, language: str, voice: str):
```

Reply:
Done, thank you @theomonnom